app/models/app_config.rb

```diff
@@ -0,0 +1,5 @@
+class AppConfig < ApplicationRecord
+  def self.find
+    first
```

Comment:
it feels a bit weird to override the `.find` method to just return `first`, why don't you just call `AppConfig.first` in the two places where you're calling `AppConfig.find`?